testpilot/experiments/serializers.py

```diff
@@ -55,6 +56,9 @@ def get_installations_url(self, obj):
                        args=(obj.pk,))
         return request.build_absolute_uri(path)
 
+    def get_survey_url(self, obj):
+        return 'https://qsurvey.mozilla.com/s3/%s' % obj.slug
```

Comment:
All of the urls provided from @MattGrimes have this structure. We can follow up with him to be sure though